src/pages/home/sidebar/SubNavigation/SubNavigation.js

```diff
@@ -17,17 +26,117 @@ const propTypes = {
+                        {
+                            icon: Expensicons.Lightbulb,
+                            text: translate('subNavigation.insights'),
+                            value: SCREENS.INSIGHTS_OLDDOT,
+                            onSelected: () => {
+                                Navigation.navigate(ROUTES.INSIGHTS_OLDDOT);
+                            },
+                        },
+                    ]}
+                />
+            </View>
+            <View style={[StyleSheet.absoluteFillObject, StyleUtils.displayIfTrue(sidebarNavigation.selectedSubNavigationMenu === CONST.SUB_NAVIGATION_MENU.WORKSPACES)]}>
+                <SubNavigationMenu
+                    title={translate('workspace.common.workspace')}
+                    menuItems={[
+                        {
+                            icon: Expensicons.User,
+                            text: translate('subNavigation.individual'),
+                            value: SCREENS.INDIVIDUAL_WORKSPACE_OLDDOT,
+                            onSelected: () => {
+                                Navigation.navigate(ROUTES.INDIVIDUALS_OLDDOT);
+                            },
+                        },
+                        {
+                            icon: Expensicons.Building,
+                            text: translate('subNavigation.groups'),
+                            value: SCREENS.GROUPS_WORKSPACES_OLDDOT,
+                            onSelected: () => {
+                                Navigation.navigate(ROUTES.GROUPS_OLDDOT);
+                            },
+                        },
+                        {
+                            icon: Expensicons.CardsDomains,
+                            text: translate('subNavigation.cardsAndDomains'),
+                            value: SCREENS.DOMAINS_OLDDOT,
+                            onSelected: () => {
+                                Navigation.navigate(ROUTES.DOMAINS_OLDDOT);
+                            },
+                        },
+                    ]}
+                />
+            </View>
+            <View style={[StyleSheet.absoluteFillObject, StyleUtils.displayIfTrue(sidebarNavigation.selectedSubNavigationMenu === CONST.SUB_NAVIGATION_MENU.DOMAINS)]}>
+                <SubNavigationMenu
+                    title={translate('globalNavigation.money')}
+                    customHeader={
+                        <HeaderWithBreadcrumbs
+                            title="Expensify.com"
+                            breadcrumbs={['Workspaces', 'Groups']}
+                            onPress={() => {
+                                // TODO: decide where to go
+                                // navigation.navigate('somwhere')
+                            }}
+                        />
+                    }
+                    menuItems={[
+                        {
+                            icon: Expensicons.Receipt,
+                            text: 'tmp1',
```

Comment:
Testing?

Reply:
Because I can't test the iframe with real olddot I decided to prepare an example submenu so someone could use that as an example.

Given that we don't have to make it on time for money2020 I think I will try to resolve the problem with cookies and finish implementation by myself. I think this answer will be the same for a few lines in this PR that also look like placeholders.